build.gradle.kts

```diff
@@ -44,10 +46,17 @@ repositories {
 
 val licenseHeaderFile = file("gradle/license-header.txt")
 spotless {
+    lineEndings = com.diffplug.spotless.LineEnding.UNIX
```

Comment:
Without this line I locally get:
```
> Task :spotlessKotlinCheck FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':spotlessKotlinCheck'.
> The following files had format violations:
      src\compatTest\kotlin\io\github\gradlenexus\publishplugin\BaseGradleTest.kt
          @@ -1,60 +1,60 @@
          -/*\n
          - * Copyright 2021 the original author or authors.\n
          - *\n
          - * Licensed under the Apache License, Version 2.0 (the "License");\n
          - * you may not use this file except in compliance with the License.\n
          - * You may obtain a copy of the License at\n
          - *\n
          - *      http://www.apache.org/licenses/LICENSE-2.0\n
          - *\n
          - * Unless required by applicable law or agreed to in writing, software\n
          - * distributed under the License is distributed on an "AS IS" BASIS,\n
          - * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n
          - * See the License for the specific language governing permissions and\n
          - * limitations under the License.\n
          - */\n
          -\n
          -package io.github.gradlenexus.publishplugin\n
          -\n
          -import org.assertj.core.api.Assertions.assertThat\n
          -import org.gradle.testkit.runner.BuildResult\n
          -import org.gradle.testkit.runner.BuildTask\n
          -import org.gradle.testkit.runner.GradleRunner\n
          -import org.gradle.testkit.runner.TaskOutcome\n
          -import org.junit.jupiter.api.io.TempDir\n
          -import java.io.File\n
          -\n
          -abstract class BaseGradleTest {\n
          -\n
          -    private val gradleRunner = GradleRunner.create()\n
          -        .withPluginClasspath()\n
          -\n
          -    @TempDir\n
          -    protected lateinit var projectDir: File\n
          -\n
          -    protected fun run(vararg arguments: String): BuildResult {\n
          -        return gradleRunner(*arguments).build()\n
          -    }\n
          -\n
          -    protected fun gradleRunner(vararg arguments: String): GradleRunner {\n
          -        return gradleRunner\n
          -//                .withDebug(true)\n
          -            .withProjectDir(projectDir)\n
          -            .withArguments(*arguments, "--stacktrace")\n
          -            .forwardOutput()\n
          -    }\n
          -\n
          -    protected fun BuildResult.assertSuccess(taskPath: String) {\n
          -        assertSuccess { it.path == taskPath }\n
      ... (72 more lines that didn't fit)
  Violations also present in 28 other files.
  Run 'gradlew.bat :spotlessApply' to fix these violations.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 26s
3 actionable tasks: 2 executed, 1 up-to-date
```
which is strange considering the CI runs `build` on `windows-latest` CI. I have a feeling `git` auto-crlf is in play.

Reply:
Trying https://github.com/gradle-nexus/publish-plugin/pull/167/commits/e03db0da926e9d34d118ab5f306eef304477d198 to hopefully resolve this issue.